packages/storage/src/bucket.js

```diff
@@ -627,6 +627,72 @@ Bucket.prototype.deleteFiles = function(query, callback) {
   });
 };
 
+
+/**
+ * Delete one or more labels from this bucket.
+ *
+ * @param {string=|string[]=} labels - The labels to delete. If no labels are
+ *     provided, all of the labels are removed.
+ * @param {function} callback - The callback function.
+ * @param {?error} callback.err - An error returned while making this request.
+ * @param {object} callback.metadata - The bucket's metadata.
+ *
+ * @example
+ * //-
+ * // Delete all of the labels from this bucket.
+ * //-
+ * bucket.deleteLabels(function(err, metadata) {});
+ *
+ * //-
+ * // Delete a single label.
+ * //-
+ * bucket.deleteLabels('label', function(err, metadata) {});
+ *
+ * //-
+ * // Delete a specific set of labels.
+ * //-
+ * bucket.deleteLabels(['label', 'labeltwo'], function(err, metadata) {});
```

Comment:
To be consistent with the unit test below, maybe we should use 

bucket.deleteLabels(['labelone', 'labeltwo']